overviews.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+mkdir -p overviews
+curl -o overviews/de_overpass.jpg "https://i.imgur.com/VHWIjSB.jpeg"
+curl -o overviews/de_mirage.jpg "https://i.imgur.com/6HLCCp1.jpeg"
+curl -o overviews/de_vertigo.jpg "https://i.imgur.com/q8HmCHN.jpeg"
+curl -o overviews/de_vertigo_lower.jpg "https://i.imgur.com/qCbHmJ2.jpeg"
+curl -o overviews/de_nuke.jpg "https://i.imgur.com/frbUQtK.jpeg"
+curl -o overviews/de_nuke_lower.jpg "https://i.imgur.com/vaF05M5.jpeg"
+curl -o overviews/de_cache.jpg "https://i.imgur.com/KqtNKUc.jpeg"
+curl -o overviews/de_inferno.jpg "https://i.imgur.com/MW7Kp7f.jpeg"
+curl -o overviews/de_train.jpg "https://i.imgur.com/FSLMR6V.jpeg"
+curl -o overviews/de_dust2.jpg "https://i.imgur.com/OrsYdDq.jpeg"
+curl -o overviews/de_ancient.jpg "https://i.imgur.com/2ZWUOt3.jpeg"
```

Comment:
Maybe could get the overviews from https://github.com/zoidbergwill/csgo-overviews ? It seems to be a nice repo to rely on for this and probably easier to get updated maps.
What do you think about it?

Reply:
yes, I've updated the script